package.json

```diff
@@ -123,10 +126,14 @@
     "react-test-renderer": "16.14.0",
     "sass-loader": "10.1.1",
     "shelljs": "0.8.4",
+    "source-map-loader": "^2.0.0",
     "stylelint": "13.9.0",
     "stylelint-config-prettier": "8.0.2",
     "stylelint-config-wordpress": "17.0.0",
-    "webpack": "4.46.0",
-    "webpack-cli": "4.4.0"
+    "ts-jest": "^26.5.0",
+    "ts-loader": "^8.0.14",
+    "typescript": "^4.1.3",
+    "webpack": "^5.20.1",
+    "webpack-cli": "4.5.0"
```

Comment:
I needed to update webpack because TypeScript required a newer version. We're now using the latest version of webpack.

This had the side effect of making it necessary to update `@wordpress/dependency-extraction-webpack-plugin`, which introduced some soon-to-come deprecations that I'm not sure how to fix. The warning is visible when running webpack, e.g. via `npx webpack`.

I think we need to change how we use `WordPressExternalDependenciesPlugin` in `webpack.config.js`, I'm just not sure what changes are required.

Reply:
That's a great step forward, comparing to my attempts to update webpack version, great job!
Let's create a new ticket for the deprecation - it goes beyond this PR scope.

Reply:
Issue opened in #1287